### PR TITLE
Advances over Login component

### PR DESCRIPTION
**Pull request by MFRamon**

### CODE REVIEW

File: src/components/FloatingActionButton.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+const FloatingActionButton = () => {
+
+    const classes = useStyles();
+
+    return (
+        <Fragment>
+
+            <Fab color="#D7D7D7" variant="extended" aria-label="delete" className={classes.fab}>
+                <FontAwesomeIcon icon={faUserPlus} className="icon" className={classes.extendedIcon}/>
```
Comment (CarlosAvina) [inline src/components/FloatingActionButton.js]: You got duplicated props, className is specified twice.

File: src/components/FloatingActionButton.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const FloatingActionButton = () => {
         <Fragment>
 
             <Fab color="#D7D7D7" variant="extended" aria-label="delete" className={classes.fab}>
-                <FontAwesomeIcon icon={faUserPlus} className="icon" className={classes.extendedIcon}/>
+                <FontAwesomeIcon icon={faUserPlus} className="icon-login" className={classes.extendedIcon}/>
```
Comment (CarlosAvina) [inline src/components/FloatingActionButton.js]: Duplicated props

Comment (MFRamon) [reply src/components/FloatingActionButton.js]: Done

File: src/components/FloatingActionButton.js (hunk 1 above)
Comment (MFRamon) [reply src/components/FloatingActionButton.js]: Done

**Pull request merged**